### PR TITLE
feature/browser compatibility

### PR DESCRIPTION
**Pull request by meodai**
Proposal to make the script work in the browser as well.

I am a bit confused on who to deliver a different main faile to node or the browser any suggestions @zeke?

### CODE REVIEW

**Comment by meodai** [comment]
@zeke this is what I came up with: https://github.com/words/wikipedia-tldr/pull/6/commits/7115b73048f85323e248425af2e2ac40042ad3f2 

Its basically wrapped in a UMD function without bundling the node requirements in, since the browser already knows `fetch` and `URL`. 

The only issue left is that jest is esLinting the main file that is minified.

**Comment by zeke** [comment]
Wow, cool idea! I will trust your judgement here implementation-wise. Happy to review and help cut a release when this is closer to being ready. 🚀

**Comment by meodai** [comment]
@zeke so that would be my proposal. I ha happy to hear your thoughts. (PS: included a browser demo)

Already using it in production here :D https://parrot.color.pizza/color/ff00aa

File: index.node.js.tpl [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const fetch = require('node-fetch')
+const { URL, URLSearchParams } = require('url')
+
+{{body}}
```
Comment (dy) [inline index.node.js.tpl:4]: I wonder if there's really no other way to include js code, templating looks a bit like overkill

Comment (meodai) [reply index.node.js.tpl:4]: @dy seemed more straight forward then to include something like webpack. How else could this be done?

Comment (meodai) [reply index.node.js.tpl:4]: @zeke what are your thoughts on this?

Comment (zeke) [reply index.node.js.tpl:4]: I think it's fine. Easy to reason about. Lightweight. No new deps. 👍🏼

**Comment by zeke** [comment]
This is awesome!

Published it as a major version bump just to be on the safe side.


```
$ np major --no-2fa

Commits:
- feature/browser compatibility (#6)  6c88b62
- scrap semantic-release after many failed attempts  0d0340b
- fix: try passing main branch as CLI arg  02f8909
- fix: udpate package lock  86ad21a
- fix: configure semantic release branch  d99f924
- fix: rename default branch from master to main  83b5cd4
- change master to main  8c5451e
- fix: migrate from travis to actions, set up semantic-release  6b428df
- fix: clean up response text (and lint with standard)  2b26b13

Commit Range:
v0.0.0...master

Registry:
https://registry.npmjs.org/


  ✔ Prerequisite check
  ✔ Git
  ✔ Installing dependencies using npm
  ✔ Running tests using npm
  ✔ Bumping version using npm
  ✔ Publishing package using npm
  ✔ Pushing tags
  ✔ Creating release draft on GitHub

 wikipedia-tldr 2.0.0 published 🎉
```

**Pull request merged**